Adding bio to master branch

rjsmatmit opened the pull request:
Oh sweet!  It's **Markdown!**  I can _mark things down_ and make a list:
- [x] A topic
- [x] Another _topic_
- [x] Topic number **three**

rjsmatmit committed:
Created bio, added 3 line items

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque pulvinar mauris nec urna viverra hendrerit. Sed auctor ipsum diam. Maecenas dignissim facilisis cursus. Phasellus pellentesque porta nibh id molestie. Donec finibus eu velit eget mollis. Cras a aliquet magna, eu volutpat eros. Pellentesque sit amet nisl in lacus faucibus pretium. Donec ut urna finibus metus interdum commodo volutpat non ante. Mauris nec fringilla diam.

rjsmatmit commented:
Making another _comment_, with another **list**:
- [x] Is this showing up?
- [ ] Can I see the progress bar?
- [x] Can anyone hear me?
- [ ] ...help?

ghost commented:
@rjsmatmit  I find your use of checklists admirable, I myself have found the act of checking the checkbox  a very satisfying action.

rjsmatmit committed:
Enhanced bio with checklists

Per the suggestion of @ProxyCat -- more checklists have been added.

rjsmatmit commented:
@berthakuo  -- you have been chosen _completely at random_ to confirm that I can **merge** this with the Master Branch.
- [x] Added checklist
- [x] Added :dollar: lots of money :dollar:
- [x] Won at everything
